types/bootstrap/js/dist/carousel.d.ts

```diff
@@ -90,15 +83,15 @@ declare namespace Carousel {
          *
          * @default "hover"
          */
-        pause: string | boolean;
+        pause: 'hover' | false;
 
         /**
          * Autoplays the carousel after the user manually cycles the first item. If
          * "carousel", autoplays the carousel on load.
          *
          * @default false
          */
-        slide: string | boolean;
+        ride: 'carousel' | false;
```

Comment:
```suggestion
        ride: 'carousel' | boolean;
```
should be probably left as `boolean`, as the code uses expression evaluation:
https://github.com/twbs/bootstrap/blob/main/js/src/carousel.js#L463